.github/workflows/run-performance.yaml

```diff
@@ -23,26 +23,113 @@ on:
+          yq e '.metadata.name = env(THIS_TEST)' performance/$TEST_NAME/network.yaml \
+            > performance/$THIS_TEST/network.yaml
+          yq e '.spec.ceramic[0].image = env(JS_CERAMIC_IMAGE)' -i performance/$THIS_TEST/network.yaml
+          yq e '.spec.ceramic[0].ipfs.rust.image = env(RUST_CERAMIC_IMAGE)' -i performance/$THIS_TEST/network.yaml
+          cat performance/$THIS_TEST/network.yaml
+          echo "THIS_TEST=$THIS_TEST" >> $GITHUB_ENV
+          echo "THIS_TEST_NAMESPACE=keramik-${THIS_TEST}" >> $GITHUB_ENV
+          echo "TEST_NAME=${TEST_NAME}" >> $GITHUB_ENV
+
+      - name: Setup GKE auth
+        uses: 'google-github-actions/auth@v1'
+        with:
+          credentials_json: ${{ secrets.GKE_SA_KEY }}
+
+      - name: Get GKE credentials
+        uses: 'google-github-actions/get-gke-credentials@v1'
+        with:
+          cluster_name: ${{ vars.GKE_CLUSTER }}
+          location: ${{ vars.GKE_ZONE }}
+
+      - name: Deploy network
+        run: |
+          kubectl apply -f performance/$THIS_TEST/network.yaml
+
+      - name: Wait for bootstrap to complete
+        timeout-minutes: 8
+        run: |
+          set -exo pipefail
+          sleep 60
+          kubectl wait --for=condition=ready \
+            --timeout=240s \
+            pod \
+            -l app=ceramic \
+            -n ${THIS_TEST_NAMESPACE}
+          sleep 60
+          kubectl wait --for=condition=complete \
+            --timeout=120s \
+            job/bootstrap \
+            -n ${THIS_TEST_NAMESPACE}
+
+      - name: Template simulation
+        run: |
+          set -exo pipefail
+          yq e '.metadata.namespace = env(THIS_TEST_NAMESPACE)' performance/$TEST_NAME/simulation.yaml \
+            > performance/$THIS_TEST/simulation.yaml
+          cat performance/$THIS_TEST/simulation.yaml
+
+      - name: Run simulation
+        run: |
+          set -exo pipefail
+          kubectl apply -f performance/$THIS_TEST/simulation.yaml
+          SIMULATION_RUNTIME=$(yq e '.spec.runTime' performance/$THIS_TEST/simulation.yaml)
+          echo "SIMULATION_RUNTIME=${SIMULATION_RUNTIME}" >> $GITHUB_ENV
+
+      - name: Notify Discord
+        env:
+          SIMULATION_COLOR: 3066993
+        run: |
+          set -exo pipefail
+          export SIMULATION_STATUS_TAG="Simulation $THIS_TEST started"
+          export CLUSTER_NAME=${{ vars.GKE_CLUSTER }}
+          envsubst < notifications/notification-template.json  > message.json
+          cat message.json
+          curl -v -H "Content-Type: application/json" -X POST -d @./message.json "${{ secrets.DISCORD_WEBHOOK_URL_SUCCEEDED }}"
+
+      - name: Wait for simulation to complete
+        run: |
+          set -exo pipefail
+          # runtime is in minutes
+          sleep $((${SIMULATION_RUNTIME} * 60))
+          kubectl wait --for=condition=complete \
+            --timeout=120s \
+            job/simulate-manager \
+            -n ${THIS_TEST_NAMESPACE}
```

Comment:
I left result processing out of this for now since it will be different for each scenario, yeah?

We can definitely add another step to process and disseminate results later.

Reply:
It doesn't have to be, we can have a red-green signal definition that can be logged or exported for each scenario. You're right, it does require more thought. We can add a ticket for this and cover it later?